src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -600,6 +600,12 @@ export const duplicateForm = (
     overrideProps.endPage = omit(originalForm.endPage, 'buttonLink')
   }
 
+  // if MRF, set submissionLimit (i.e. response limit) = null
```

Comment:
/nit might want to explain why we are doing this instead

Mrf doesn't support submission limit